<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<qualifiedTypeNameHistroy>
    <fullyQualifiedTypeName name="java.io.IOException"/>
    <fullyQualifiedTypeName name="java.lang.Error"/>
    <fullyQualifiedTypeName name="java.lang.NegativeArraySizeException"/>
    <fullyQualifiedTypeName name="com.nov15.task.IllegalStateExceptionDemo"/>
    <fullyQualifiedTypeName name="com.nov15.task.ExceptionHandlingDemo"/>
    <fullyQualifiedTypeName name="com.nov18.task.DivisonExample"/>
    <fullyQualifiedTypeName name="javax.security.sasl.SaslException"/>
    <fullyQualifiedTypeName name="java.util.List"/>
    <fullyQualifiedTypeName name="java.util.stream.Stream"/>
    <fullyQualifiedTypeName name="com.nov18.task.AreaCalculator"/>
    <fullyQualifiedTypeName name="java.lang.IllegalArgumentException"/>
    <fullyQualifiedTypeName name="com.nov18.task.ClassCastExceptionDemo"/>
    <fullyQualifiedTypeName name="java.lang.ClassCastException"/>
    <fullyQualifiedTypeName name="java.lang.Class"/>
    <fullyQualifiedTypeName name="com.nov18.task.ClassNotFoundExpDemo"/>
    <fullyQualifiedTypeName name="java.lang.AutoCloseable"/>
    <fullyQualifiedTypeName name="java.io.Closeable"/>
    <fullyQualifiedTypeName name="com.r.try_with_resource.DatabaseResource"/>
    <fullyQualifiedTypeName name="com.r.try_with_resource.FileResource"/>
    <fullyQualifiedTypeName name="java.lang.IncompatibleClassChangeError"/>
    <fullyQualifiedTypeName name="java.lang.NullPointerException"/>
    <fullyQualifiedTypeName name="java.lang.NumberFormatException"/>
    <fullyQualifiedTypeName name="java.util.InputMismatchException"/>
    <fullyQualifiedTypeName name="java.lang.ArithmeticException"/>
    <fullyQualifiedTypeName name="java.lang.ArrayIndexOutOfBoundsException"/>
    <fullyQualifiedTypeName name="java.util.Spliterator"/>
    <fullyQualifiedTypeName name="java.lang.ArrayStoreException"/>
    <fullyQualifiedTypeName name="java.lang.ClassNotFoundException"/>
    <fullyQualifiedTypeName name="java.lang.Throwable"/>
    <fullyQualifiedTypeName name="java.lang.Exception"/>
    <fullyQualifiedTypeName name="com.r.myexception.InvalidAgeException"/>
    <fullyQualifiedTypeName name="java.lang.RuntimeException"/>
    <fullyQualifiedTypeName name="com.r.myexception.GreaterMarksException"/>
    <fullyQualifiedTypeName name="java.lang.InterruptedException"/>
    <fullyQualifiedTypeName name="java.lang.Override"/>
    <fullyQualifiedTypeName name="java.util.Collection"/>
    <fullyQualifiedTypeName name="java.util.Collections"/>
    <fullyQualifiedTypeName name="java.util.Vector"/>
    <fullyQualifiedTypeName name="com.r.vector.Employee"/>
    <fullyQualifiedTypeName name="java.lang.Byte"/>
    <fullyQualifiedTypeName name="java.lang.Short"/>
    <fullyQualifiedTypeName name="java.lang.Long"/>
    <fullyQualifiedTypeName name="java.lang.Float"/>
    <fullyQualifiedTypeName name="java.lang.Double"/>
    <fullyQualifiedTypeName name="java.lang.FunctionalInterface"/>
    <fullyQualifiedTypeName name="com.r.lambda_expression.Accept"/>
    <fullyQualifiedTypeName name="java.lang.Integer"/>
    <fullyQualifiedTypeName name="java.lang.String"/>
    <fullyQualifiedTypeName name="java.util.Scanner"/>
    <fullyQualifiedTypeName name="java.lang.System"/>
    <fullyQualifiedTypeName name="java.util.function.Predicate"/>
    <fullyQualifiedTypeName name="java.util.function.Consumer"/>
    <fullyQualifiedTypeName name="com.r.functional_interface_demo.Employee"/>
</qualifiedTypeNameHistroy>
